conf/proxy.conf

```diff
@@ -74,3 +74,6 @@ tlsCertificateFilePath=
 
 # Path for the TLS private key file
 tlsKeyFilePath=
+
+# Validates hostname when proxy creates tls connection with broker
+isTlsHostnameVerificationEnable=false
```

Comment:
-> `isTlsHostnameVerificationEnabled` ?
or 
`enableTlsHostnameVerification`

Reply:
as all the tls related config has tls prefix so, I have renamed it to `tlsHostnameVerificationEnabled`